readthedocs/doc_builder/python_environments.py

```diff
@@ -295,19 +295,31 @@ def venv_path(self):
         return os.path.join(self.project.doc_path, 'envs', self.version.slug)
 
     def setup_base(self):
-        site_packages = ''
+        """
+        Create a virtualenv, invoking python -m virtualenv
```

Comment:
```suggestion
        Create a virtualenv, invoking python -m virtualenv.
```

The linter will fail bc of this

Reply:
Thanks -- also impressive to be faster than the linter :runner: :rocket:  :)